Update Amcrest:markdown

jamesarbrown opened the pull request:
## Proposed change
Following new code mechanisms by @pnbrucker the scan_interval is now obsolete as the camera polling has been replaced by a new event mechanism. Also added a note that the new mechanism does not work with a certain block of FW.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

jamesarbrown committed:
Update Amcrest:markdown

Following new code mechanisms by @pnbrucker the scan_interval is now obsolete as the camera polling has been replaced by a new event mechanism. Also added a note that the new mechanism does not work with a certain block of FW.

frenck commented on source/_integrations/amcrest.markdown:
```diff
@@ -92,7 +92,7 @@ authentication:
   type: string
   default: basic
-scan_interval:
+scan_interval: (Obsolete from 108.5)
```
This breaks the build

jamesarbrown replied:
Is there a deprecated format for this? 

Would this work?

```
scan_interval

    (integer)(Optional)(Before Version 108.5)

    Defines the update interval of the sensor in seconds.

    Default value:

    10

```

klaasnicolaas replied:
Just put it in the description?

frenck commented on source/_integrations/amcrest.markdown:
```diff
@@ -143,6 +143,11 @@ Newer Amcrest firmware may not work, then `rtsp` is recommended instead.
 make sure to follow the steps mentioned at [FFmpeg](/integrations/ffmpeg/)
 documentation to install the `ffmpeg`.
 
+**Note:** From 108.5 a new motion detection mechanism has been implemented for Amcrest cameras
+to improve speed response, whilst lowering network demands. Some cameras have a 
+propreitary bug in the firmware (V2.420.0009.0.R.20151106) and will return a 500 Bad Response
+error and are incompatible with Motion Detection.
```
Small typo. Furthermore, maybe we should use the infoboxes we have for this:

<https://developers.home-assistant.io/docs/documentation_create_page#html>

```suggestion
**Note:** From 108.5 a new motion detection mechanism has been implemented for Amcrest cameras
to improve speed response, whilst lowering network demands. Some cameras have a 
proprietary bug in the firmware (V2.420.0009.0.R.20151106) and will return a 500 Bad Response
error and are incompatible with Motion Detection.
```

frenck reviewed (changes requested):
Thanks for the PR @jamesarbrown!

Left a comment, could you please take a look? 👍

klaasnicolaas commented:
something for the alert website? @frenck

frenck commented:
I honestly don't know. This does not affect a Home Assistant version, but a specific firmware version from the vendor... Would be an endless alert? 🤷

jamesarbrown committed:
Update source/_integrations/amcrest.markdown

Co-Authored-By: Franck Nijhof <frenck@frenck.nl>

jamesarbrown committed:
Update amcrest.markdown

Amended the scan_interval description to include notice that from 108.5 it has no effect

frenck commented on source/_integrations/amcrest.markdown:
```diff
@@ -93,7 +93,7 @@ authentication:
   type: string
   default: basic
 scan_interval:
-  description: Defines the update interval of the sensor in seconds.
+  description: Defines the update interval of the sensor in seconds. This setting is fixed from HA version 108.5 and scan_interval has no effect
```
This is a bit weird? According to the code it is still there?

If it has no effect, a bug should be raised.

jamesarbrown replied:
Hi Frenck,

Please see comment by @pnbruckner
https://github.com/home-assistant/core/issues/33875#issuecomment-617189318

But it seems although this was closed "wontfix"
https://github.com/tchellomello/python-amcrest/issues/155

That @pnbrucker has had a change of heart
https://github.com/home-assistant/core/pull/36955

So this markdown needs dropping.

James

pnbruckner commented:
For "the record", the `scan_interval` configuration option stopped having an effect when the integration was changed long ago from "platform based" sensors to "hub based." This was before my time. I made an attempt to "fix" that (see architecture issue [#194](https://github.com/home-assistant/architecture/issues/194)), but the approach was not approved.
